src/Option_Command.php

```diff
@@ -436,6 +436,100 @@ public function update( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Gets the 'autoload' value for an option.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : The name of the option to get 'autoload' of.
+	 *
+	 * @subcommand get-autoload
+	 */
+	public function get_autoload( $args ) {
+		global $wpdb;
+
+		list( $option ) = $args;
+
+		$existing = $wpdb->get_row(
+			$wpdb->prepare(
+				"SELECT autoload FROM $wpdb->options WHERE option_name=%s",
+				$option
+			)
+		);
+		if ( ! $existing ) {
+			WP_CLI::error( "Could not get '{$option}' option. Does it exist?" );
+
+		}
+		WP_CLI::log( $existing->autoload );
+	}
+
+	/**
+	 * Sets the 'autoload' value for an option.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : The name of the option to set 'autoload' for.
+	 *
+	 * <autoload>
+	 * : Requires WP 4.2. Should this option be automatically loaded.
```

Comment:
Why would this require 4.2? As far as I understand, 4.2 added the `$autoload` arg to `update_option()`, but we're not using that here. Given the `autoload` DB field was introduced with 3.4 (or even earlier?), this should work with all WP version supported by WP-CLI, shouldn't it?

Reply:
```suggestion
	 * : Should this option be automatically loaded.
```

@schlessera You are correct! I originally started to use `update_option()`, hence the comment, but had to write my own implementation because `update_option()` bails early if the value isn't changing.